testsuite/MDAnalysisTests/analysis/test_distances.py

```diff
@@ -19,18 +19,21 @@
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
+
+
 from __future__ import print_function, absolute_import
 
 import pytest
 import scipy
 import scipy.spatial
+from math import *
```

Comment:
In general don't use `import *` ever, because it makes it unclear where a function came from.  Also, use `np.sqrt` not math